lib/logging.dart

```diff
@@ -61,6 +61,10 @@ class Logger {
     return _loggers.putIfAbsent(name, () => new Logger._named(name));
   }
 
+  factory Logger.orphan(String name, {Logger parent, Map<String, Logger> children: const {}}) {
```

Comment:
One more question. I'm wondering whether to keep the `parent` and `children` parameters. Do you need the detached loggers to be hierarchical? If not, maybe we can just remove both parameters.

If you do need support for hierarchical loggers, then I would suggest 2 changes:
* document that you shouldn't give a parent that is a permanent logger (like root), because then it would not be detached as you intended. 
* remove `const` from the default value of `children` and add type arguments, since we still want to allow adding children later (similar to how we do this in Logger._named).